packages/entities/entities-consumer-groups/package.json

```diff
@@ -64,7 +64,7 @@
     "extends": "../../../package.json"
   },
   "distSizeChecker": {
-    "errorLimit": "400KB"
+    "errorLimit": "410KB"
```

Comment:
@Leopoldthecoder | @adamdehaven Had to bump this slightly, the build was unexpectedly failing otherwise:

https://github.com/Kong/public-ui-components/actions/runs/6897551205